### PR TITLE
added missing license statements

### PR DESCRIPTION
**Pull request by jrgnicho**
Fix for issue #8

### CODE REVIEW

File: kinematics_tests_support/package.xml [hunk 1]
```diff
@@ -3,48 +3,10 @@
...
-  <!-- One license tag required, multiple allowed, one license per tag -->
-  <!-- Commonly used license strings: -->
-  <!--   BSD, MIT, Boost Software License, GPLv2, GPLv3, LGPLv2.1, LGPLv3 -->
-  <license>TODO</license>
-
-
-  <!-- Url tags are optional, but mutiple are allowed, one per tag -->
-  <!-- Optional attribute type can be: website, bugtracker, or repository -->
-  <!-- Example: -->
-  <!-- <url type="website">http://wiki.ros.org/kinematics_tests_support</url> -->
```
Comment (davetcoleman) [inline kinematics_tests_support/package.xml:22]: do you mind adding this or something similar:

```
  <url type="website">http://moveit.ros.org</url>
  <url type="bugtracker">https://github.com/ros-planning/moveit/issues</url>
  <url type="repository">https://github.com/ros-planning/moveit</url>
```

?

File: kinematics_base_test/src/test_kinematics_plugin.cpp [hunk 2]
```diff
@@ -1,3 +1,39 @@
+/*********************************************************************
+ * Software License Agreement (BSD License)
+ *
+ *  Copyright (c) 2015, Southwest Research Institute
```
Comment (davetcoleman) [inline kinematics_base_test/src/test_kinematics_plugin.cpp:4]: Your call - but do you want this to be 2017 instead?

File: kuka_kr210_manipulator_ik_plugin/package.xml [hunk 3]
```diff
@@ -3,41 +3,16 @@
...
-  <!--   <build_depend>message_generation</build_depend> -->
-  <!-- Use buildtool_depend for build tool packages: -->
-  <!--   <buildtool_depend>catkin</buildtool_depend> -->
-  <!-- Use run_depend for packages you need at runtime: -->
-  <!--   <run_depend>message_runtime</run_depend> -->
-  <!-- Use test_depend for packages you need only for testing: -->
-  <!--   <test_depend>gtest</test_depend> -->
+  <maintainer email="jrgnichodevel@gmail.com">Jorge Nicho</maintainer>
+  <license>BSD</license>
+  <author email="jrgnichodevel@gmail.com">Jorge Nicho</author>
```
Comment (davetcoleman) [inline kuka_kr210_manipulator_ik_plugin/package.xml]: same here for website address

File: motoman_sia20d_ikfast_manipulator_plugin/package.xml [hunk 4]
```diff
@@ -3,39 +3,16 @@
...
-  <!--   <build_depend>message_generation</build_depend> -->
-  <!-- Use buildtool_depend for build tool packages: -->
-  <!--   <buildtool_depend>catkin</buildtool_depend> -->
-  <!-- Use run_depend for packages you need at runtime: -->
-  <!--   <run_depend>message_runtime</run_depend> -->
-  <!-- Use test_depend for packages you need only for testing: -->
-  <!--   <test_depend>gtest</test_depend> -->
+  <maintainer email="jrgnichodevel@gmail.com">Jorge Nicho</maintainer>
+  <license>BSD</license>
+  <author email="jrgnichodevel@gmail.com">Jorge Nicho</author>
```
Comment (davetcoleman) [inline motoman_sia20d_ikfast_manipulator_plugin/package.xml]: same here for website address

**Review by davetcoleman — CHANGES_REQUESTED** [review]

File: kinematics_base_test/src/test_kinematics_plugin.cpp (hunk 2 above)
Comment (jrgnicho) [reply kinematics_base_test/src/test_kinematics_plugin.cpp:4]: The file was created in 2015 if I recall correctly, I'm not sure if there are any rules on this but I went on date of creation.

**Comment by jrgnicho** [comment]
@davetcoleman sorry for the delay on this, just made the changes you requested.

**Review by davetcoleman — APPROVED** [review]

**Comment by davetcoleman** [comment]
@jrgnicho the CI failed because of broken tests, I think from the previous PR. This PR is fine to merge since it does not change code but I'd really appreciate you looking at the broken test.

**Comment by jrgnicho** [comment]
@davetcoleman thanks for approving, I'll look into that.

**Pull request merged**